src/depot.ts

```diff
@@ -124,7 +124,8 @@ export async function build(inputs: Inputs) {
     ...flag('--lint', inputs.lint),
     ...flag('--lint-fail-on', inputs.lintFailOn),
     ...flag('--save', inputs.save),
-    ...flag('--save-tag', inputs.saveTag)
+    ...flag('--save-tag', inputs.saveTag),
+    ...flag('--save-tag', inputs.saveTags),
```

Comment:
does this mean that if someone specifies both saveTag and saveTags, they will all be used?